test/device_map_dialog_test.rb

```diff
@@ -0,0 +1,69 @@
+require_relative "test_helper"
+
+require "bootloader/device_map_dialog"
+
+describe Bootloader::DeviceMapDialog do
+  # just simple tests to avoid typos as logic is not so easy to test
+  describe "#run" do
+    before do
+      Yast.import "BootStorage"
+
+      device_map = Bootloader::DeviceMap.new(
+        "/dev/sda" => "hd0",
+        "/dev/sdb" => "hd1"
+      )
+      allow(Yast::BootStorage).to receive(:DisksOrder)
+        .and_return(device_map.disks_order)
+    end
+
+    def mock_ui_events(*events)
+      allow(Yast::UI).to receive(:UserInput).and_return(*events)
+      allow(Yast::UI).to receive(:QueryWidget).and_return("/dev/sda")
+      allow(Yast::UI).to receive(:OpenDialog).and_return(true)
+      allow(Yast::UI).to receive(:CloseDialog).and_return(true)
+    end
+
+    it "always returns symbol :back" do
+      mock_ui_events(:ok)
+
+      expect(subject.run).to eq :back
```

Comment:
I agree. But I have been adding the parenthesis so far because it's how we always do it. It's a grey area at https://github.com/SUSE/style-guides/blob/master/Ruby.md#method-calls

If we all agree, I'd happily stop using parenthesis with ```eq```, I just want to be sure that is common feeling.